lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,178 @@
-          }
-
-          > a {
-            border: 1px solid $border-color-1;
-            box-shadow: 0 1px 0 rgba(255, 255, 255, .35) inset;
-            background: $sidebar-active-image;
-
-            &:after {
-              opacity: 1.0;
-              right: 15px;
-            }
+    // reseting bolded fonts for the course index
+    .group-heading {
+        @extend %t-regular;
+    }
 
-            p {
-              color: #333;
+    // we're targetting the .class now, instead of the #id
+    .accordion {
+        @extend %t-copy-sub1;
+
+        .course-navigation {
+
+            .button-chapter {
+                @include transition(all $tmg-s3 ease-in-out);
+                @include box-sizing(border-box);
+                @include linear-gradient(top, $sidebar-chapter-bg-top, $sidebar-chapter-bg-bottom);
+                @include transition(background-color .1s linear 0s);
+                width: 100% !important;
+                border: 0;
+                border-radius: 0;
+                padding: 0;
+                box-shadow: 0 1px 0 $white inset, 0 -1px 0 $shadow-l1 inset;
+                background-color: $sidebar-chapter-bg;
+                color: $link-color;
+                -webkit-font-smoothing: subpixel-antialiased; // this brings back our nice, dark blue
+
+                .group-heading {
+                    @extend %t-title6;
+                    position: relative;
+                    display: block;
+                    padding: ($baseline*.75) $baseline ($baseline*.75) ($baseline*2);
+                    @include text-align(left);
+
+                    .icon {
+                        position: absolute;
+                        @include left($baseline);
+                        top: $baseline;
+                        color: $gray;
+                    }
+                }
+
+                &.is-open {
+                    background: $white;
+                    box-shadow: none;
+                }
+
+                &.active {
+
+                    .group-heading {
+                        @extend %t-strong;
+                        color: $base-font-color;
+
+                        .icon {
+                            color: $gray-d1;
+                        }
+                    }
+                }
+
+                &:hover,
+                &:focus {
+                    background: $white;
+                }
+
+                &:active {
+                    outline: none;
+                }
             }
-          }
 
-          span.subtitle {
-            @extend %t-weight2;
-          }
-        }
-
-        &.graded {
-          > a {
-            .icon {
-              vertical-align: middle;
+            .chapter-content-container {
+                background: $white;
+
+                &:focus {
+                    // we don't need the blue glow to be here at this point
+                    // although focus is sent for correct keyboard/a11y
+                    outline: none;
```

Comment:
We have this technically, just with a different name (`.sr-is-focusable`). It's a bit different here though because this container contains something whereas the other doesn't. I think it's fine to be a one-off thing here. If we use this method again in the future we can revisit.